docs/self-managed/modeler/web-modeler/configuration/configuration.md

```diff
@@ -1,81 +1,24 @@
-| `SPRING_DATASOURCE_URL` | [optional]<br/>Can be used to provide a customized connection string (for example, to enable SSL-secured connections).<br/>If set, `RESTAPI_DB_HOST`, `RESTAPI_DB_PORT`, and `RESTAPI_DB_NAME` don't need to be provided. | `jdbc:postgresql://postgres.example.com:5432/modeler-db?sslmode=require` |
-
-#### Configuring SSL for the database connection
-
-The generic way to configure an SSL connection between Web Modeler and the database is as follows:
-
-- Modify database connection URL `SPRING_DATASOURCE_URL` and customize connection parameters.
-- Provide SSL certificates and keys to the `restapi` component, if required.
-
-Consult the [PostgreSQL documentation](https://jdbc.postgresql.org/documentation/ssl/) for a description
-of the different SSL modes and the security provided.
-
-For a full list of all available connection parameters, visit the [PostgreSQL documentation](https://jdbc.postgresql.org/documentation/use/#connection-parameters/).
-
-Below are examples for common scenarios, increasing in the level of security they provide.
-
-**SSL mode "require"**
-
-In this mode, an SSL connection is established between Web Modeler and the database. This mode is still prone
-to man-in-the-middle attacks.
-
-- Modify database URL: `jdbc:postgresql://[DB_HOST]:[DB_PORT]/[DB_NAME]?sslmode=require`
-
-No certificates are needed in Web Modeler for this mode.
-
-**SSL mode "verify-full"**
-
-In this mode, Web Modeler requests a certificate from the database server to verify its identity. This mode is not
-prone to man-in-the-middle attacks.
-
-To enable this mode, mount the root certificate with which the server certificate was signed.
-
-- Provide root certificate at this location: `myCA.crt -> ~/.postgresql/root.crt`
-- Modify database URL: `jdbc:postgresql://[DB_HOST]:[DB_PORT]/[DB_NAME]?ssl=true`
-
-**SSL mode "verify-full" with client certificates**
-
-In this mode, Web Modeler requests a certificate from the database server to verify the server's identity, and
-the server requests a certificate from the client to verify the client's identity.
-
-To enable this mode, mount the client certificates.
-
-- Provide client certificates at these locations:
-  - `myClientCertificate.pk8 -> ~/.postgresl/postgresql.pk8`
-  - `myClientCertificate.crt -> ~/.postgresl/postgresql.crt`
-- Provide root certificate at this location: `myCA.crt -> ~/.postgresql/root.crt`
-- Modify database URL: `jdbc:postgresql://[DB_HOST]:[DB_PORT]/[DB_NAME]?ssl=true`
-
-Furthermore, configure the database server to verify client certificates:
-[PostgreSQL documentation](https://www.postgresql.org/docs/current/ssl-tcp.html)
+Please refer to the [Advanced Database Configuration Guide](./database.md) for details on how to configure Web Modeler's database connection.
```

Comment:
@jfriedenstab I decided against repeating the table with environment variables for configuring the database connection to not run into duplication issues like in #2047.

@christinaausley From a technical writer perspective, is it fine to transfer some parts of the configuration to sub-pages or would this make the guide harder to understand?

Reply:
> Hm, but you wouldn't need to repeat the table, would you?

Ah true, haven't thought about that. I now moved the table to the "Overview" page again 👍